### ISSUE TITLE
Confused about the label z.

### ISSUE THREAD

**Issue by vigor95** [tensorflow/minigo#305]
It seems the z label(winner of a game) is directly black(+1) or white(-1) in this implementation. But in the original paper(Mastering the Game of Go without Human Knowledge), z is the winner from the perspective of the current player at step t. It means if the current player is black and white wins finally, z at this step will be -1.
Am I wrong or this implementation also works?

**Comment by sethtroisi** [comment]
Hey Vigor, 

Andrew and I talked briefly about this. You are totally correct that we don't seem to be labeling the data the same as the paper. We appreciate you noticing this (and taking the time to tell us).

It's not clear if both implementations learn the same thing. We have a small test setup (starting from the initial state running training over X million examples) that we can test on but unfortunately it's not a top priority right now, I'll update this thread when I get time to run it.

**Comment by Ttl** [comment]
ELF used the same fixed labeling and it worked pretty well for them. I don't think this is a big issue.
